README.md

```diff
@@ -14,3 +14,6 @@ to deploy the site on your host locally:
 bundle install --path vendor/bundle
 bundle exec jekyll serve --incremental
 ```
+
+## Additional Licenses
+MIT License for the getting started page: Copyright (c) 2009-2019 Bitcoin.org.
```

Comment:
I'd rather us not have a bitcoin.org license in our source code for the sake of that getting started page. It might take a little bit longer to do, but given how simple this design is, we might as well just build it up from scratch? Happy to take a stab at it.

Reply:
There is a ton of css so not sure I have the bandwidth to address this right now. Feel free to take a stab at it (also check the responsiveness too see screen cap below):
<img width="615" alt="Capture d’écran 2019-11-05 à 14 10 42" src="https://user-images.githubusercontent.com/11842328/68237897-190edf00-ffd6-11e9-818b-7baa0a48bd95.png">
Right now I'd advocate to merge this as is and removing the Bitcoin.org components later.